### ISSUE TITLE
SQL Panel DeprecationWarning - The "default.html" templates for forms and formsets will be removed

### ISSUE THREAD

**Issue by bluetech** [jazzband/django-debug-toolbar#1813]
When I open the SQL Panel on Django 4.2, django-debug-toolbar 4.1, I get the following warning:

> django.utils.deprecation.RemovedInDjango50Warning: The "default.html" templates for forms and formsets will be removed. These were proxies to the equivalent "table.html" templates, but the new "div.html" templates will be the default from Django 5.0. Transitional renderers are provided to allow you to opt-in to the new output style now. See https://docs.djangoproject.com/en/4.2/releases/4.1/ for more details

I tried to narrow down a bit what's causing it using pdb, but I only managed to see that the `self` is `SignedDataForm`, not much more.

**Comment by living180** [comment]
I looked at this awhile back, and AFAICT, the warning is actually generated by an internal compatibility shim in Django.  It should go away once the compatibility shim is removed.  Although the Debug Toolbar could be updated to avoid the warning, any changes would be unnecessary once targeting Django 5.0.  Thus I decided that it wasn't worth code churn on the Debug Toolbar end.  If anyone would like more details, let me know, and I can try to gather them together in the next few days.
